examples/repl.rs

```diff
@@ -63,98 +65,97 @@ fn prepare_home_dir() -> PathBuf {
 fn main() {
     env_logger::init();
 
-    let app = cli::make_cli_subcommands();
-    let mut repl_app = app.clone().setting(AppSettings::NoBinaryName);
-
-    let app = cli::add_global_flags(app);
-
-    let matches = app.get_matches();
+    let cli_opt: WalletOpt = WalletOpt::from_args();
 
     // TODO
-    // let level = match matches.occurrences_of("v") {
-    //     0 => LevelFilter::Info,
-    //     1 => LevelFilter::Debug,
+    // let level = match cli_opt.verbosity.map(|v| v.len()) {
```

Comment:
Is there any chance this is `None` here? I would map the info level to `None | Some(0)` just in case..

Reply:
I changed this around a bit to take a log level name, defaulting to "info". It may be possible to do the `-vv` style, but I didn't see an obvious way to do it with StructOpt, and using `-l error` style is a bit more explicit.